src/main/resources/assets/securitycraft/lang/en_us.json

```diff
@@ -428,15 +428,15 @@
 	"help.securitycraft.wire_cutters.info": "Rightclicking a mine with the wire cutters will defuse it.",
 	"help.securitycraft.keypad_item.info": "The key panel is used in the crafting recipes of keypads, password-protected chests, and password-protected furnaces.",
 	"help.securitycraft.admin_tool.info": "The admin tool's main purpose is to display information about a SecurityCraft block. It will show the owner's UUID and name, and other information, like passcodes and any inserted modules.",
-	"help.securitycraft.taser.info": "The taser is a self-recharging stun gun. Rightclick to fire it. Entities hit with the taser will get Nausea II, Slowness II, and Weakness II for 10 seconds. If the Taser is sneak-rightclicked (provided some redstone in the inventory) it will double its strength for the next shot.",
-	"help.securitycraft.camera_monitor.info": "The camera monitor is a handheld screen, used to view live camera feeds. Rightclick a security camera to view it on the monitor. Sometimes, it may automatically shutoff to save resources, simply rightclick to enable it again (configurable).",
-	"help.securitycraft.sc_manual.info": "The SecurityCraft Manual shows information about all the blocks and items SecurityCraft adds.",
+	"help.securitycraft.taser.info": "The taser is a self-recharging stun gun. Rightclick to fire it. Entities hit with the taser will get Nausea II, Slowness II, and Weakness II for 10 seconds. If the Taser is sneak-rightclicked (provided some redstone in the inventory), it will double its strength for the next shot.",
+	"help.securitycraft.camera_monitor.info": "The camera monitor is a portable item used to view live feeds of up to 30 bound security cameras. Right-click a security camera with the camera monitor to bind it, then right-click air and select the camera in the GUI while the camera’s chunk is loaded to see its feed. Unbind a security camera by right-clicking it again with the camera monitor or by opening the camera monitor's GUI and clicking the corresponding \"X\" button.",
+	"help.securitycraft.sc_manual.info": "The SecurityCraft Manual shows basic information regarding all the blocks and items SecurityCraft adds. For more in-depth descriptions, check out the FTB wiki for SecurityCraft.",
```

Comment:
Could always provide a link to SecurityCraft's homepage on the FTB wiki here so that users don't have to Google it.

Reply:
https://ftb.gamepedia.com/Category:SecurityCraft :P

Reply:
Uhhh, I would go with the second one since it's easier to type and it also has links to the other SC items on that page as well (once I update the navbox :P).

Reply:
Can hyperlinks be inserted in the manual? That’d also be good for the Block Pocket image.

Reply:
No they can't. I've tried recently, but I'd have to know the place that the link text is at in order to make it clickable and I haven't come up with a proper solution to that yet.

Reply:
The location on the screen? Is there a method to return coordinates or something like that to find the first letter, then add getStringLength (or whatever Eclipse’s version is)?

Reply:
Yeah, there are ways to insert links into chat messages but it's not as easy to add links to strings in GUIs. Just typing out the link will be fine then.

Reply:
Maybe a "Type /FTB (/BlockPocket) to get a hyperlink" then?

Reply:
If anything, `/sc ftb` and `/sc blockpocket`.

Reply:
Right. Up to you guys if you want to create those.

Reply:
From my side, not for now. I still want to revisit the clickable text since I had a few ideas that I haven't been able to try yet.

Reply:
kk. Will reword entry as 

> "The SecurityCraft Manual shows basic information regarding all the blocks and items SecurityCraft adds. For more in-depth descriptions, check out the FTB wiki at https://ftb.gamepedia.com/SecurityCraft."